docling/models/factories/__init__.py

```diff
@@ -0,0 +1,25 @@
+import logging
+from functools import lru_cache
+
+from docling.models.factories.ocr_factory import OcrFactory
+from docling.models.factories.picture_description_factory import (
+    PictureDescriptionFactory,
+)
+
+logger = logging.getLogger(__name__)
+
+
+@lru_cache()
+def get_ocr_factory(allow_external_plugins: bool = False):
```

Comment:
should we not define the return types?

Reply:
since the `lru_cache` is wrapping the function, it actually makes sense